spyder/app/mainwindow.py

```diff
@@ -738,7 +743,7 @@ def _update_shortcuts_in_panes_menu(self, show=True):
                     try:
                         context = '_'
                         name = 'switch to {}'.format(section)
-                        shortcut = CONF.get_shortcut(
+                        shortcut = self._conf.get_shortcut(
```

Comment:
Maybe a new method call `get_conf_shortcut` should be implemented which returns `self._conf.get_shorcut` in a similar way as done for `set_conf` and `get_conf`?